src/awkward/_connect/numba/arrayview_cuda.py

```diff
@@ -0,0 +1,35 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+
+from numba import types
+
+import awkward as ak
+
+########## ArrayView Arguments Handler for CUDA JIT
+
+
+class ArrayViewArgHandler:
```

Comment:
It's fine to import this in `ak.numba._register` because it doesn't load anything CUDA-specific (won't fail on computers with GPUs or environments without `cudatoolkit`).

I don't understand, however, how `numba.cuda.jit` knows that this `ArrayViewArgHandler` belongs to the `ArrayView` class—I don't see any registration making it so. Is it by name?

No complaint; I'm just saying that I don't know how `numba.cuda.jit` _finds_ this class, though it evidently does because the tests would fail if it didn't.